### PR TITLE
Ser 2019 maize recommendation 2

### PR DESCRIPTION
**Pull request by Bahatiroben**
Kenya impact trainings:
1. maize recommendations
you can review the telerivet's actions code from here
https://telerivet.com/p/0c6396c9/service/SVe6ca9e74be559552

and check the functionality with a kenya number.
https://telerivet.com/p/0c6396c9/service/SVbc04709b5abb48c0/edit
- dial in the short code
- add account number: 26215997
- on the second screen, choose maize recommendation on option 11
 you should receive 2 kickoff messages and be able to send the response to the asked questions.

### CODE REVIEW

**Review by nyabongo — CHANGES_REQUESTED** [review]
Assuming this is meant to be [SER-219](https://oneacrefund.atlassian.net/browse/SER-219), the test service you provide is configured as manually triggered, shouldn't it be an automatically triggered service like https://telerivet.com/p/0c6396c9/service/SV59d10a657b3e5119/edit
i.e instead of  `When manually triggered for a contact:` shouldn't it be `When incoming or outgoing call is received or answered:`

File: TEST-RESULTS.xml [hunk 1]
```diff
@@ -0,0 +1,1375 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nyabongo) [inline TEST-RESULTS.xml:1]: can we not add this back to the repo?

File: shared/rosterApi/registerClient.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+var Log = require('../../logger/elk/elk-logger');
```
Comment (nyabongo) [inline shared/rosterApi/registerClient.js:1]: Is this file neccessary for this ticket?

**Review by nyabongo — DISMISSED** [review]

File: TEST-RESULTS.xml (hunk 1 above)
Comment (Bahatiroben) [reply TEST-RESULTS.xml:1]: okay.

File: shared/rosterApi/registerClient.js (hunk 2 above)
Comment (Bahatiroben) [reply shared/rosterApi/registerClient.js:1]: Ohhh might have got it from another branch while rebasing.

**Review by shejasabin — APPROVED** [review]
The code looks good. I did not test it though

**Comment by Bahatiroben** [comment]
> The code looks good. I did not test it though

The test requires a Kenyan simcard on a real device. so it was done by @nyabongo

**Review by DevFojo — APPROVED** [review]
LGTM

**Pull request merged**